frontend/apps/reader/modules/readerdictionary.lua

```diff
@@ -136,6 +136,7 @@ function ReaderDictionary:init()
                 f:close()
                 local dictname = content:match("\nbookname=(.-)\r?\n")
                 local is_html = content:find("sametypesequence=h", 1, true) ~= nil
+                local lang_from, lang_to, _, _ = content:match("lang=(%a+)-?(%a*)\r?\n?")
```

Comment:
you can remove the `_, _` (would they be filled ? and if they were, it's fine if there's no var to receive them in Lua.)

Reply:
I was thinking of string.find, which returns the indexes of the match as well. Not that it really makes sense here :sweat_smile: 
I've removed it